client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -240,7 +243,7 @@
 </template>
 
 <script>
-import { CleanHtml, DpFlyout, DpInlineNotification } from '@demos-europe/demosplan-ui'
+import { CleanHtml, DpFlyout, DpInlineNotification, DpContextualHelp } from '@demos-europe/demosplan-ui'
```

Comment:
can be removed if not used.

Reply:
thanks done https://github.com/demos-europe/demosplan-core/pull/2832/commits/0e05eed35480ba1137ca30bb5d4eea0ca477e2ab